.github/workflows/Upgrade-CI.yml

```diff
@@ -0,0 +1,285 @@
+name: Upgrade - CI
+on:
+  pull_request_target:
+    types: [ opened, labeled, unlabeled, synchronize ]
+    branches:
+      - 'main'
+    paths-ignore:
+      - 'LICENSE'
+      - '**/.gitignore'
+      - '**.md'
+      - '**.adoc'
+      - '*.txt'
+      - '.github/**'
+      - 'kustomize/**'
+      - 'dev/**'
+      - 'app-interface/**'
+jobs:
+  event-bridge-build:
+    env:
+      ORIGINAL_BRANCH: main
```

Comment:
What about using here target branch instead of `main`? In 99% of the cases it is the same but it is probably safer.

Reply:
Yes, you are right. It will be more general. Done.